FrameworkSystem/scripts/dirac-populate-component-db.py

```diff
@@ -43,15 +44,25 @@ def setExcludedHosts( value ):
 
 componentType = ''
 
+result = getProxyInfo()
+if not result[ 'OK' ]:
+  gLogger.error( result[ 'Message' ] )
+  DIRACexit( -1 )
+
+# If the user is not a ServiceAdministrator the script will not work
+if not 'ServiceAdministrator' in result['Value']['groupProperties']:
+  gLogger.error( 'ServiceAdministrator property missing' )
+  DIRACexit( -1 )
+
```

Comment:
Hi Andrei, the access rule is enforced also in the service. The fact it is added also here is to prevent the script to try and connect to 50+ hosts just to tell for everyone that it can't access.